### ISSUE TITLE
SD clock generation

### ISSUE THREAD

**Issue by harbaum** [WangXuan95/FPGA-SDcard#8]
What is the intended sd card clock rate?

From the comments, it seems that it's supposed to stay below 25Mhz as CLK_DIV is adjusted to one for up to 25Mhz, to 2 for 25 to 50 Mhz and so on ...

However, the resulting clock is much lower. Clockdivr is clockdiv + 1 and clkcnt counts to { clkdivr, 1}. With CLK_DIV set to 2 the counter runs from 0 to 7, effectively dividing the system clock by 8. So on a 32Mhz system, the SD card clock is 4 Mhz. Is this intended that way?

**Comment by WangXuan95** [comment]
Yes, this is a intended design, without reason, just because I didn't calculate it carefully. After all, if the SD clock is too fast, it won't work, but if it is slow, it will still work (although it will affect performance)
